client/bower.json

```diff
@@ -32,7 +32,9 @@
     "paper-avatar": "Abe90/paper-avatar#^1.0.4",
     "web-animations-js": "^2.3.1",
     "paper-dropdown": "https://github.com/elauria/paper-dropdown.git#^2.0.1",
-    "iron-autogrow-textarea": "^2.1.0"
+    "iron-autogrow-textarea": "^2.1.0",
+    "opentracing-javascript-utils": "https://github.com/zalando-zmon/opentracing-javascript-utils.git#^1.0.0",
+    "lightstep-overlay": "https://github.com/lightstep/lightstep-overlay.git"
```

Comment:
Can we remove vendor specific libs from OSS and may be find a way to include in the browser post-deployment? May be via a config and using a CDN link?

Reply:
yes! just keeping it here for now to see it working but thanks for reminding me!